utils/questions.js

```diff
@@ -43,6 +43,16 @@ const createAppQuestions = [
   }
 ];
 
+const featureQuestions = [
+  {
+    type: 'checkbox',
+    message: 'Select optional features you want to add (Press enter to skip)',
+    name: 'features',
+    choices: []
```

Comment:
Are we adding choices later?

Reply:
These are just default options and overriding it in index.js at line 317 and 365.